docs/user-guide/tool-integration/nuget-chocolatey-example.md

```diff
@@ -0,0 +1,305 @@
+
+``` tab="Windows" linenums="1"
+choco search -s "%REPO_URL%"
+``` 
+
+``` tab="Linux" linenums="1"
+choco search -s "$REPO_URL"
+```
+
+The output should be like follows:
+```log
+Chocolatey <version>
+2 validations performed. 1 success(es), 0 warning(s), and 0 error(s).
+
+hello-chocolatey 1.0.0
+1 packages found.
+```
+
+### Delete a Chocolatey package from strongbox
+
+Chocolatey does not have a built in way to delete packages from the server. 
+As a workaround, you can use `nuget` to delete packages, see [hello-strongbox-nuget-visual-studio] for more details.
+
+### Install a package
+
+``` tab="Windows" linenums="1"
+# Execute the following command as a administrator!
+choco install hello-chocolatey
+```
+
+``` tab="Linux" linenums="1"
+choco install hello-chocolatey
+```
+
+The output should be like follows:
+
+```log
+Chocolatey <version>
+2 validations performed. 1 success(es), 0 warning(s), and 0 error(s).
+
+Installing the following packages:
+hello-chocolatey.1.0.0.nupkg
+By installing you accept licenses for the packages.
+
+hello-chocolatey v1.0.0
+hello-chocolatey package files install completed. Performing other installation steps.
+Package would install here
+ The install of hello-chocolatey was successful.
+  Software install location not explicitly set, could be in package or
+  default install location if installer.
+
+Chocolatey installed 1/1 packages.
+ See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
+```
+
+### Uninstall a package
+
+``` tab="Windows" linenums="1"
+# Execute the following command as a administrator!
+choco uninstall hello-chocolatey
+```
+
+``` tab="Linux" linenums="1"
+choco uninstall hello-chocolatey
+```
+
+The output should be like follows:
+```log
+Chocolatey <version>
+2 validations performed. 1 success(es), 0 warning(s), and 0 error(s).
+
+Uninstalling the following packages:
+hello-chocolatey
+
+hello-chocolatey v1.0.0
+ Skipping auto uninstaller - No registry snapshot.
+ hello-chocolatey has been successfully uninstalled.
+
+Chocolatey uninstalled 1/1 packages.
+ See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
+```
+
+## See also
+
+* [Chocolatey commands reference][choco-reference]
+* [Chocolatey docs][choco-docs]
+* [hello-strongbox-nuget-chocolatey]
+
+[choco]: https://chocolatey.org/
+[choco-reference]: https://chocolatey.org/docs/commands-reference
+[choco-docs]: https://chocolatey.org/docs
+[choco-repo]: https://github.com/chocolatey/choco
+[example files]: https://github.com/strongbox/strongbox-examples/tree/master/hello-strongbox-nuget-chocolatey
+[mono]: https://www.mono-project.com/
+[official documentation]: https://chocolatey.org/install
+[strongboxci/alpine]: https://hub.docker.com/r/strongboxci/alpine/tags?page=1&name=choco
+[strongbox-examples]: https://github.com/strongbox/strongbox-examples
+[hello-strongbox-nuget-chocolatey]: https://github.com/strongbox/strongbox-examples/tree/master/hello-strongbox-nuget-chocolatey
```

Comment:
Points to broken link.

Reply:
That is expected since it's not merged into `master` yet :)